.gitignore

```diff
@@ -1,3 +1,6 @@
+# env files with production access
+product-service/env/
```

Comment:
normally you should use wildcards, like *.env

Reply:
Yeah, for node variables it is the right way to use the .env files, but in my case i moved the variables declaration from serverless yml file to keep all the secret data within the separate file and de able to add it to gitignore file.